### PR TITLE
WT-9032 Convert wt2853_perf test to a performance test

### PR DESCRIPTION
**Pull request by jiechenbo**
Convert the correctness test on WT-2853_perf to be a performance test instead.

### CODE REVIEW

**Comment by jiechenbo** [comment]
Hey @ddanderson I have given my best efforts to try convert the test to a performance test. I have also tried write some of the comments but I have a lack of knowledge around the functionality of the test. Let me know what you think of the test 😄

File: test/evergreen.yml [hunk 1]
```diff
@@ -954,15 +968,6 @@ tasks:
         vars:
           directory: test/thread
 
-  - name: bench-wtperf-test
```
Comment (Siddhartha8899) [inline test/evergreen.yml:957]: Why do we delete this task?

Comment (jiechenbo) [reply test/evergreen.yml]: Oops! I added it back, thanks!

File: bench/wt2853_perf/main.c [hunk 2]
```diff
@@ -262,7 +262,12 @@ thread_insert(void *arg)
             flag = 0;
         }
         maincur->set_value(maincur, post, bal, extra, flag, key);
-        testutil_check(maincur->insert(maincur));
+        ret = maincur->insert(maincur);
+        if (ret == WT_ROLLBACK) {
+            testutil_check(session->rollback_transaction(session, NULL));
+            continue;
+        }
+        testutil_assert(ret == 0 || ret == WT_ROLLBACK);
```
Comment (ddanderson) [inline bench/wt2853_perf/main.c:270]: You can just assert that `ret == 0`.  `ret` cannot be `WT_ROLLBACK` at this point.

**Review by ddanderson — APPROVED** [review]
Looks good.  I noted two potential improvements, take them if you like.

File: test/evergreen.yml [hunk 3]
```diff
@@ -601,6 +610,11 @@ functions:
       params:
         file: ./wiredtiger/cmake_build/test/cppsuite/${test_name}.json
 
+  "upload wt2853_perf test stats":
```
Comment (Siddhartha8899) [inline test/evergreen.yml:613]: can we only have one `perf.send `function?

Comment (jiechenbo) [reply test/evergreen.yml]: Done! let me know how you think of it.

**Review by Siddhartha8899 — APPROVED** [review]
LGTM! 💯

**Pull request merged**